src/engraving/dom/measure.cpp

```diff
@@ -1930,6 +1933,16 @@ bool Measure::isFirstInSystem() const
     return system()->firstMeasure() == this;
 }
 
+//---------------------------------------------------------
+//   isFirstInSystem
+//---------------------------------------------------------
+
+bool Measure::isFirst() const
```

Comment:
To prevent confusion, should we rename this method to `isFirstInSection`?

Reply:
And match with the function header (comment)

Reply:
Good idea, thanks.